### PR TITLE
28 clean up encryptio py

### PR DESCRIPTION
**Pull request by Morn98**

### CODE REVIEW

File: encryptio.py [hunk 1]
```diff
@@ -302,7 +282,7 @@ def decrypt_blowfish(data, key, mode):
             print('This mode is not available for the selected algorithm')
             return
 
-    print(plaintext)
+    print("Decryption successfull: " + plaintext)
```
Comment (tzigiannis) [inline encryptio.py:285]: this throws a TypeError: can only concatenate str (not "bytes") to str

**Review by tzigiannis — CHANGES_REQUESTED** [review]
please remove the error. the rest looks good 🌮

File: encryptio.py (hunk 1 above)
Comment (Morn98) [reply encryptio.py]: Resolved this problem by encoding plaintext bytes to utf-8.

Comment (tzigiannis) [reply encryptio.py]: Looks good 👍

**Pull request merged**